.github/workflows/release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Make release with Build binaries
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build-binary:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: PyInstaller win64
+        id: win64
+        uses: action-python/pyinstaller-py3.8@win64
+        with:
+          path: ./
+          requirements: "./requirements_nt.txt"
+      - name: PyInstaller amd64
```

Comment:
What to do with `build-binary` job of `Build Artifacts` workflow? Is it actual or we can remove it and even simplify the Dockerfile?

Reply:
The Build Artifacts process has been removed and the Dockerfile build process has been simplified